src/functions/deleteTestResult/application/delete-test-result-service.ts

```diff
@@ -7,12 +7,12 @@ export const deleteTestResult = async (): Promise<void> => {
   const connection: mysql.Connection = getConnection();
   try {
     const [testResultDeleted] = await connection.promise().query(deleteTestResultRecord());
-    info('No of TEST_RESULT records Deleted: ', testResultDeleted.affectedRows);
+    info('No of TEST_RESULT records Deleted: ', testResultDeleted['affectedRows']);
```

Comment:
Just want to check, as the `mysql` package has been upgraded, has the shape of the payload from the query changed or have tighter types been added?

Reply:
as suggested added interface and removed this change from all effected services